spec/spec_helper.rb

```diff
@@ -2,5 +2,6 @@
 Coveralls.wear!
 RSpec.configure do |config|
   config.before { allow($stdout).to receive(:puts) }
-  config.after(:each) { DebtCeiling.configure {|c| c.debt_ceiling = nil; c.reduction_target = nil; c.reduction_date = nil } }
+  config.after(:each) { DebtCeiling.configure {|c| c.debt_ceiling = nil; c.reduction_target = nil; c.reduction_date = nil } ; DebtCeiling.clear }
+  config.after(:all) { DebtCeiling.calculate('.') }
```

Comment:
This is debt ceiling going all inception/ouroboros...  does it count as self-hosting when it's part of its own test suite?  I dunno, but it can fail for having too much tech debt now.  This also serves as a super simple example of how easy it is to integrate into any test suite.